tests/test_adaptor_neighbour_list.cc

```diff
@@ -35,6 +35,7 @@
 namespace rascal {
 
   BOOST_AUTO_TEST_SUITE(neighbour_list_adaptor_test);
+  // TODO(felix) add adaptor test for the no center option
```

Comment:
Yes, that needs to be done before merge, otherwise I fear it will be forgotten.

Reply:
it is tested in adaptor_strict, forgot to rm the comment